tutorials/cloud-iot-rtdp/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Real time Data Processing With Google Cloud IoT
+description: Demonstrate how to build data processing backend for Google Cloud IoT.
+author: teppeiy
+tags: Cloud IoT Core, Cloud Dataflow, Cloud Functions, Java, Node.js
+date_published: 2017-11-29
+---
+# Real time Data Processing With Google Cloud IoT
+This repository contains sample code for Real time Data Processing With Google Cloud IoT solution. Sample MQTT client generates simulated data and submits them to Cloud IoT Core. Cloud Functions and Cloud Dataflow receive data through Cloud Pub/Sub.
```

Comment:
Agreed. I think the user would benefit from more detailed description of the solution. I'm curious why this is going into the community site instead of a more formal tutorial.

Reply:
This tutorial will be referred from a solution article under "https://cloud.google.com/solutions/ " so more detailed description will be available there. Also, I will update README to refer back to the article once it's available.